src/Zynga/Framework/IO/Web/V1/Manager.hh

```diff
@@ -20,64 +20,18 @@ class Manager {
    * @throws Zynga\Framework\IO\Web\V1\Exception\WebIO
    */
   public static function putFile(UrlBox $uploadUrl, string $fileName, string $mimeType): void {
-    $ch = static::initializeCurl($uploadUrl);
-    if ($ch === false) {
-      throw new FailedInitializationException('Failed to initialize cURL handle');
-    }
-
-    $cfile = static::createCurlFile($fileName, $mimeType, $fileName);
-    $data = array('file' => $cfile);
-    static::setCurlOption($ch, CURLOPT_POSTFIELDS, http_build_query($data));
-    static::setCurlOption($ch, CURLOPT_CUSTOMREQUEST, "PUT");
+    $returnCodes = array();
+    $exec = "curl -s -I '".$uploadUrl->get()."' --upload-file '$fileName' | grep HTTP/1.1 | awk {'print $2'}";
```

Comment:
Talked offline, try using curl's --write-out %{http_code} instead of the grep HTTP/1.1 | awk stuff

Reply:
How about 
```
exec($cmd, $output, $returnCode);
    if ($returnCode !== 0) {
```